requirements/gateway-requirements.txt

```diff
@@ -3,7 +3,13 @@
 # dev/generate_requirements.py script to regenerate this file.
 
 pydantic<3,>=1.0
-fastapi<1
+# fastapi 0.109.2 upgraded their `starlette` dependency,
+# which has a breaking interaction with `slowapi` when
+# the ".env" file doesn't exist.
+# https://github.com/laurentS/slowapi/pull/184 has been
+# filed to address this, but until it is approved we need
+# to pin the fastapi dependency
+fastapi<0.109.1
```

Comment:
i saw i'm supposed to edit `dev/generate_requirements` to update this, but it doesn't allow us to specify a max patch version. i can use `unsupported` in the YAML file but then we have to update this with every single new fastapi release.

Reply:
oh nevermind there is CI forcing us to edit the YAML file